src/jit/x64/emit.dasc

```diff
@@ -2084,6 +2086,23 @@ void MVM_jit_emit_primitive(MVMThreadContext *tc, MVMJitCompiler *compiler, MVMJ
         | call FUNCTION;
         break;
     }
+    case MVM_OP_sp_getlexstatic_o: {
+        MVMint16 dst       = ins->operands[0].reg.orig;
+        MVMint16 name      = ins->operands[1].reg.orig;
+        MVMint16 spesh_idx = ins->operands[2].lit_i16;
+        MVMuint32 slot     = ins->operands[3].lit_ui32;
+        | mov ARG1, TC;
+        | get_spesh_slot ARG2, spesh_idx;
+        | lea ARG2, STATICFRAME:ARG2->body.inline_cache;
```

Comment:
Doesn't this overwrite the spesh slot fetched into ARG2?

Reply:
Ah, yes it does and that's ok, because you're just doing what I suggested :D

Reply:
Ha, yep.